app/services/error-handler.js

```diff
@@ -6,27 +6,22 @@ export default class ErrorHandlerService extends Service {
 
 @service l10n;
 
-handleError(error) {
+handle(error) {
   const errorStatus =  (error?.errors?.[0]?.status) ? parseInt(error?.errors?.[0]?.status) : error;
+  const errorMsg = error?.errors?.[0]?.detail ? error?.errors?.[0]?.detail : error;
   switch (errorStatus) {
     case 403:
+      this.notify.error(this.l10n.tVar(errorMsg));
```

Comment:
We either show errorMsg or predefined message. In case errorMsg is defined, we show it, else predefined message